firebase/functions/src/index.ts

```diff
@@ -1,25 +1,11 @@
 import * as admin from 'firebase-admin';
 import * as functions from 'firebase-functions';
 
-import { TriggerAnnotated, Event } from 'firebase-functions';
-import { DeltaSnapshot } from 'firebase-functions/lib/providers/database';
-
 export { postInvocationWrite } from './post-invocation-write';
 export { postLabWrite } from './post-lab-write';
 export { postExecutionWrite } from './post-execution-write';
 export { bucketChange } from './uploads/add-outputs';
 export { postUserCreate } from './post-user-create';
 export { postHandshakeCommitCreate } from './post-handshake-write';
 
-const conf = functions.config().firebase;
-
-const serviceAccount = functions.config().fb_service_account;
-
-if (serviceAccount) {
-  conf.credential = admin.credential.cert(<any>{
-    private_key: serviceAccount.private_key,
-    client_email: serviceAccount.client_email
-  });
-}
-
-admin.initializeApp(conf);
+admin.initializeApp();
```

Comment:
Config is gone in 1.0.0

Reply:
How is the service account stuff supposed to work without that config?

Reply:
I assume `admin` pulls it out itself from `firebase`

Reply:
So to be more specific: `functions.config().firebase` is gone (which is what we've used here). Here are the docs: https://firebase.google.com/docs/functions/beta-v1-diff?utm_source=email&utm_medium=email&utm_campaign=cloud_functions_v1.0#functionsconfigfirebase_removed

However, you can still access firebase config via `process.env.FIREBASE_CONFIG`. I assume that's what `admin` does behind the scenes. Notice that all other config values still exists (and need to be mocked in unit tests): https://firebase.google.com/docs/functions/unit-testing#mocking_config_values

Reply:
> but we definitely need to make sure we check that to not break the CLI.

Totally. Notice that this PR is far from ready. This needs to be tested across all packages.